rspace/src/main/scala/coop/rchain/rspace/HotStore.scala

```diff
@@ -10,8 +10,10 @@ import coop.rchain.shared.Cell
 import coop.rchain.shared.MapOps._
 import coop.rchain.rspace.internal._
 import coop.rchain.rspace.Serialize._
+import coop.rchain.rspace.trace.Consume
 import scodec.Codec
 
+import scala.collection.SortedSet
```

Comment:
Why are those imports needed if you are only removing things?

Reply:
done